secure/unprivileged-building-of-container-images-using-buildah.adoc

```diff
@@ -6,15 +6,27 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Running {pipelines-shortname} as the root user on a container can expose the container processes and the host to other potentially malicious resources. You can reduce this type of exposure by running the workload as a specific non-root user in the container. To run builds of container images using Buildah as a non-root user, you can perform the following steps:
+Running {pipelines-shortname} as the root user on a container can expose the container processes and the host to other potentially malicious resources. You can reduce this type of exposure by running the workload as a specific non-root user in the container.
+
+In most cases, creating a custom task for building the image and configuring user namespaces in this task enables running Buildah without root privileges.
+
+If your image does not build successfully using this configuration, you can define custom service account (SA) and security context constraint (SCC); however, in this option, you must enable the Buildah step to raise its privileges (`allowPrivilegeEscalation: true`).
+
+include::modules/op-nonroot-buildah-user-namespaces.adoc[leveloffset=+1]
+
+[id="buildah-nonroot-sa-scc"]
+== Running Buildah as a non-root user by defining a custom SA and SCC
+
+To run builds of container images using Buildah as a non-root user, you can perform the following steps:
 
 * Define custom service account (SA) and security context constraint (SCC).
 * Configure Buildah to use the `build` user with id `1000`.
 * Start a task run with a custom config map, or integrate it with a pipeline run.
```

Comment:
outside the scope of your changes and this PR, but this list should probably be ordered if they are to be followed sequentially:
```suggestion
. Define a custom service account (SA) and security context constraint (SCC).
. Configure Buildah to use the `build` user with id `1000`.
. Start a task run with a custom config map, or integrate it with a pipeline run.
```

Reply:
Outside the scope indeed - I would not want to touch this part right now as any reworking is likely to require a separate review cycle.